vehicles/Steering/chassis.cc

```diff
@@ -360,9 +379,14 @@ void chassisTask(void* arg) {
 
   }
 }
+
+#define EXPECTED_FLAG_SUMMARY 0b11111111
```

Comment:
```suggestion
#define EXPECTED_FLAG_SUMMARY 0xFF
```
We usually use hex over binary for bit mask because distinguishing 0b111111111 and 0b11111111 is hard but 0x1FF and 0xFF is easy.

Also for `BITMAP_SUMMARY`